index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title> Leslie Portfolio project</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body class="body">
+    <header class="header">
+      <nav class="navigate">
+        <a href=""><img src="images/John Doe.png" alt="" /></a>
```

Comment:
- [ ] According to the project template, the logo and the navbar are aligning in a way where there is spaces around them.
front, back, bottom and top. 👇 

compare to what you have in you project: 👇